extensions/gleam/src/gleam.rs

```diff
@@ -37,7 +37,13 @@ impl GleamExtension {
             os = match platform {
                 zed::Os::Mac => "apple-darwin",
                 zed::Os::Linux => "unknown-linux-musl",
-                zed::Os::Windows => "pc-windows-msvc",
+                zed::Os::Windows => {
+                    if cfg!(target_env = "msvc") {
```

Comment:
I don't think these will work this way, as these extensions are compiled to WASM, and this `target_env` will never pass.

Reply:
this is only about gleam or anywhere where I put this change?

Reply:
Anything that’s an extension. I don’t think the Rust or Deno LSPs are extensions yet.

Reply:
ok, from msys2 side we will maintain this change downstream